plugins/woocommerce/includes/admin/class-wc-admin-post-types.php

```diff
@@ -124,14 +124,14 @@ public function post_updated_messages( $messages ) {
 
 		$messages['product'] = array(
 			0  => '', // Unused. Messages start at index 1.
-			/* translators: %s: Product view URL. */
-			1  => sprintf( __( 'Product updated. <a href="%s">View Product</a>', 'woocommerce' ), esc_url( get_permalink( $post->ID ) ) ),
+			/* translators: %1$s: Product link opening tag. %2$s: Product link closing tag.*/
+			1  => sprintf(__('Product updated. %1$sView Product%2$s', 'woocommerce'), '<a id="woocommerce-product-updated-message-view-product__link" href="' . esc_url(get_permalink($post->ID)) . '">', '</a>'),
```

Comment:
The formatting is off here... see the PHPCS errors.

You should make sure your dev env is setup to auto-format files when saving, to ensure that code style issues are handled for you.